package.json

```diff
@@ -22,7 +22,11 @@
     "prebuild": "yarn run clean",
     "build": "babel src -d dist",
     "clean": "del dist",
-    "test": "eslint ."
+    "lint": "eslint .",
+    "test": "yarn run lint && jest"
+  },
+  "resolutions": {
+    "babel-core": "7.0.0-bridge.0"
```

Comment:
link to a guide on how this works with the 6->7 transition?

Reply:
(in description as well: https://github.com/facebook/jest/issues/6913)